canonical-web-and-design/react-components
# Build Slider component.

## Done

- Migrated Slider component from maas-ui
- Added stories for default + with number input

## QA

- Run `yarn docs`
- Check that there is a Slider component that you can interact with.
